tests/web/test_main.py

```diff
@@ -116,14 +116,15 @@ def test_get_file_invalid_path(project_tmp_path: Path) -> None:
     config = project_tmp_path / "config.py"
     config.write_text(
         """from sqlmesh.core.config import Config
-config = Config(ignore_patterns=["*.txt"])
+    config = Config(ignore_patterns=["*.txt"])
```

Comment:
Don't change this. It's writing a Python file and the indentation makes it invalid Python.